buildSrc/src/main/kotlin/publish.gradle.kts

```diff
@@ -105,7 +105,6 @@ afterEvaluate {
                         developer {
                             id.set("cortinico")
                             name.set("Nicola Corti")
-                            email.set("corti.nico@gmail.com")
```

Comment:
I removed this as I don't think it's mandatory for MavenCentral publishing and some users might not want to disclose their email adress